lms/djangoapps/discussion_api/forms.py

```diff
@@ -2,12 +2,31 @@
 Discussion API forms
 """
 from django.core.exceptions import ValidationError
-from django.forms import BooleanField, CharField, Form, IntegerField, NullBooleanField
+from django.forms import (
+    BooleanField,
+    CharField,
+    Field,
+    Form,
+    IntegerField,
+    MultipleHiddenInput,
+    NullBooleanField,
+)
 
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.locator import CourseLocator
 
 
+class TopicIdField(Field):
+    """
+    Field for a list of topic_ids
+    """
+    widget = MultipleHiddenInput
+
+    def __init__(self, *args, **kwargs):
+        self.field_type = CharField()
```

Comment:
This sets the widget we want the generic field type to be.  But now that I think about it, it might be possible to subclass CharField and then set the MultipleHiddenInput class attribute.